man/glue_col.Rd

```diff
@@ -27,7 +29,7 @@ The \link[crayon:crayon]{crayon} package defines a number of functions used to
 color terminal output. \code{glue_col()} and \code{glue_data_col()} functions provide
 additional syntax to make using these functions in glue strings easier.
 
-Using the following syntax will apply the function \code{blue} function to the text 'foo bar'.\preformatted{\{blue foo bar\}
+Using the following syntax will apply the function \code{crayon::blue()} to the text 'foo bar'.\preformatted{\{blue foo bar\}
```

Comment:
Not sure why this happened; maybe the docs haven't been rebuilt in a while? Seems good, though.